agenta-web/src/contexts/app.context.tsx

```diff
@@ -12,6 +12,9 @@ type AppContextType = {
     error: any
     isLoading: boolean
     mutate: () => void
+
+    modalInstance: any
```

Comment:
is it possible to specify more precise types here?

Reply:
Okay will add modalInstance type as HooksAPI which is the type of modal Instance returned by useModal hook.